playwright/pages/ChangePwdPage.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable radix */
+/* eslint-disable no-useless-constructor */
+/* eslint-disable prettier/prettier */
```

Comment:
Please do not siable prettier. Please resolve all prettier issues. You can use the cli tool

Reply:
done